### ISSUE TITLE
suggestion: common-intents

### ISSUE THREAD

**Issue by Stevenic** [howdyai/bot-common-keywords#3]
I wonder if there's not a corollary to this which is a set of common intents (help, stop, yes, no, etc.) that can be used to bootstrap intent recognition for a bot. Amazon has defined a bunch of [built-in intents](https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/implementing-the-built-in-intents) that could prove to be a good starting point.  I could see crowdsourcing the defining of the language models for these common intents across multiple services (Luis.ai, Wit.ai, etc) and multiple languages.  The idea being that "Intent.Help" is triggered when a user is asking for help regardless of the users spoken language and whether the developer is using Luis.ai or Wit.ai to perform language understanding.

Question is should this be a different project or an expanded scope of this project.

**Comment by benbrown** [comment]
I think it should be part of this project @Stevenic!

**Comment by Purus** [comment]
Yeah. I agree too. Having some default intents with the project will be great.
